index.js

```diff
@@ -151,6 +154,21 @@ async function register (server, options) {
     logger.info(server.info, 'server stopped')
   })
 
+  function isLoggingIgnored (options, request) {
+    return (
+      (
+        options.ignoreTags &&
+        options.ignoreTags.some(
+          (ignoreTag) => (
+            request.route.settings.tags &&
+            request.route.settings.tags.includes(ignoreTag)
+          )
+        )
```

Comment:
This is going to be slow as essentially it's two nested loops. Could you short circuit them and use a `for(;;)` instead? Also, please move `request.route.settings.tags` to its own variable.

Reply:
Revised.